docs/sources/setup-grafana/installation/debian.md

```diff
@@ -2,13 +2,10 @@
 aliases:
   - /docs/grafana/latest/installation/debian/
   - /docs/grafana/latest/installation/installation/debian/
+  - /docs/grafana/latest/setup-grafana/installation/debian/
 description: Install guide for Grafana on Debian or Ubuntu
-keywords:
-  - grafana
-  - installation
-  - documentation
-title: Install on Debian/Ubuntu
-weight: 200
+title: Install on Debian or Ubuntu
+weight: 400
```

Comment:
the weight seems off here, this is should be the first  ?

Reply:
We can order the installs however you'd like. I can make debian/ubuntu first on the list.